lib/livebook/live_markdown/import.ex

```diff
@@ -361,24 +371,31 @@ defmodule Livebook.LiveMarkdown.Import do
   defp grab_leading_comments(elems), do: {[], elems}
 
   defp notebook_metadata_to_attrs(metadata) do
-    Enum.reduce(metadata, %{}, fn
-      {"persist_outputs", persist_outputs}, attrs ->
-        Map.put(attrs, :persist_outputs, persist_outputs)
+    Enum.reduce(metadata, {%{}, []}, fn
+      {"persist_outputs", persist_outputs}, {attrs, messages} ->
+        {Map.put(attrs, :persist_outputs, persist_outputs), messages}
+
+      {"autosave_interval_s", autosave_interval_s}, {attrs, messages} ->
+        {Map.put(attrs, :autosave_interval_s, autosave_interval_s), messages}
 
-      {"autosave_interval_s", autosave_interval_s}, attrs ->
-        Map.put(attrs, :autosave_interval_s, autosave_interval_s)
+      {"hub_id", hub_id}, {attrs, messages} ->
+        if Livebook.Hubs.hub_exists?(hub_id) do
+          {Map.put(attrs, :hub_id, hub_id), messages}
+        else
+          {attrs, messages ++ ["ignoring notebook Hub with unknown id: #{inspect(hub_id)}"]}
```

Comment:
Have we decided if the ID will be gibberish or something public? Given we will sign it, I think it can be "my-company-hub", right? Because if it is gibberish, I don't think this message would help.

Reply:
The ids have prefix like `fly-` or `enterprise-` followed by some sort of identifier that should work across livebooks. But we can skip the inspect, that's also fine.

Reply:
Skip the inspect and ship it.